src/esp/sim/Simulator.cpp

```diff
@@ -95,6 +96,8 @@ void Simulator::close(const bool destroy) {
 }
 
 void Simulator::reconfigure(const SimulatorConfiguration& cfg) {
+  if (!loggingContext_)
```

Comment:
I don't see braceless if statements much in our codebase. I would hope they are taboo.